src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -352,6 +352,7 @@ private Movie GetMovie(ParsedMovieInfo parsedMovieInfo, string imdbId, SearchCri
             // TODO: Answer me this: Wouldn't it be smarter to start out looking for a movie if we have an ImDb Id?
             if (!String.IsNullOrWhiteSpace(imdbId) && imdbId != "0")
             {
+                _logger.Debug("Try matching by imdbID");
```

Comment:
Can you remove those logging entries? They are really unnecessary and will just clutter the log (especially during an rss sync with > 300 releases).